README.md

```diff
@@ -10,6 +10,23 @@ Among other things this includes:
 - derivations to build the seL4-test suite for various (simulated and phyiscal) targets
 - a generic [FOD](https://nixos.org/manual/nixpkgs/stable/#fixed-output-derivation) fetcher for [Google's repo tool](https://android.googlesource.com/tools/repo)
 
+# Caching
+
+Many of the derivations in this repository depend on huge builds, which are not cached by
+<https://cache.nixos.org/>. To spare you (and us) hours of compiling cross-compilers, the
+CI-pipeline configured for this repository pushes its binary artifacts to the
+[DLR-FT Cachix](https://app.cachix.org/cache/dlr-ft). Hence, you may use the following binary cache
+with [Cachix](https://app.cachix.org):
```

Comment:
Maybe say *how* to use that cache? `cachix use dlr-ft` seems to work.

Reply:
I was reluctant to add that, because the https://app.cachix.org/cache/dlr-ft link already contains a full description of what to do, and I'm afraid of maintaining these instructions. But, usability wise, its the clever thing to do, so I adjusted accordingly.